src/schema/payment.ts

```diff
@@ -139,27 +127,31 @@ export interface RecurringPaymentSearchParams extends SearchParams {
 }
 
 type RecurringPaymentStatus =
-  | "InProgress"
-  | "AwaitingAuthorisation"
+  | "Accepted"
+  | "AcceptedCancellationRequest"
+  | "AcceptedTechnicalValidation"
+  | "AcceptedCustomerProfile"
+  | "AcceptedFundsChecked"
+  | "AcceptedWithChange"
+  | "Pending"
   | "Rejected"
-  | "Authorised"
-  | "Revoked"
-  | "Expired"
-
-interface RecurringPayment {
```

Comment:
Where has this interface gone? Did we need to delete it?

Reply:
Added back, not sure why I removed it in the first place